src/gui/spatial/timeRecorder.js

```diff
@@ -0,0 +1,125 @@
+/*
+* Created by Valentin on 04/23/20.
+*
+* Copyright (c) 2020 PTC Inc
+* 
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+createNameSpace("realityEditor.gui.spatial.timeRecorder");
+
+realityEditor.gui.spatial.timeRecorder.sequences = {};
```

Comment:
A significant fraction of this could be local variables inside a closure but that doesn't really matter for the purposes of this PR